tests/assets/certs/iotedgehubdev-test-only.root.ca.key.pem

```diff
@@ -0,0 +1,54 @@
+-----BEGIN ENCRYPTED PRIVATE KEY-----

```

Comment:
as for the test pems, are there expire time? Does that mean the tests will fail later?

Reply:
It will expire after 3 years (which is the max allowed valid time). I will add a comment for this in related test case.
This cert is only used to test loading certs with password protection. Since we doesn't provide option to add password for generated root ca and extra cost is required to build a test utility to create password protected cert. I manually generated a cert in advance for that single test case.